wsdlgo/encoder.go

```diff
@@ -895,6 +895,7 @@ func (ge *goEncoder) genValidator(w io.Writer, typeName string, r *wsdl.Restrict
 
 func (ge *goEncoder) genGoStruct(w io.Writer, d *wsdl.Definitions, ct *wsdl.ComplexType) error {
 	if ct.Abstract {
+		fmt.Fprintf(w, "type %s interface{}\n\n", name)
```

Comment:
Fixed. That'll teach me to rush through when I should have waited until later in the day when I had more time. Confirmed it compiles now.